### PR TITLE
docs for kubeconfig expiring tokens

### PR DESCRIPTION
**Pull request by kinarashah**

### CODE REVIEW

File: content/rancher/v2.x/en/cluster-admin/cluster-access/ace/_index.md [hunk 1]
```diff
@@ -13,6 +13,11 @@ This kubeconfig file and its contents are specific to the cluster you are viewin
 
 After you download the kubeconfig file, you will be able to use the kubeconfig file and its Kubernetes [contexts](https://kubernetes.io/docs/reference/kubectl/cheatsheet/#kubectl-context-and-configuration) to access your downstream cluster.
 
+_Available as of v2.4.6_ 
+
+If [kubeconfig expiring tokens is enabled](../api-tokens/#enabling-kubeconfig-expiring-tokens), the kubeconfig file requires [rancher cli](../cli) to be present in your PATH.
```
[the remark this answers is not in the document]
Comment (janeczku) [reply content/rancher/v2.x/en/cluster-admin/cluster-access/ace/_index.md:18]: > If kubeconfig expiring tokens is enabled

Isn't expiration always enabled now and all the admin can do is configure the TTL ? Or can he disable expiration by setting TTL to `0` ? This should be clarified.

File: content/rancher/v2.x/en/api/api-tokens/_index.md [hunk 2]
```diff
@@ -19,11 +20,35 @@ Here is the complete list of tokens that are generated with `ttl=0`:
...
+
+### Enabling Kubeconfig expiring tokens
+
+1. Go to setting kubeconfig-generate-token in the Rancher API view at `https://<Rancher-Server-IP/v3/settings/kubeconfig-generate-token`. 
+
+2. Set the setting value to `false`. 
+
+### Expiration period
+
+Expiration period or scope of kubeconfig tokens is decided by kubeconfig token ttl. By default, kubeconfig-token-ttl-minutes is 16 hours.
```
[the remark this answers is not in the document]
Comment (kinarashah) [reply content/rancher/v2.x/en/api/api-tokens/_index.md]: Yeah, I've combined the steps of disabling the setting and changing TTL under `Setting TTLs for kubeconfig tokens` so it's not a separate section that can be followed by itself

File: content/rancher/v2.x/en/api/api-tokens/_index.md [hunk 3]
```diff
@@ -19,11 +20,29 @@ Here is the complete list of tokens that are generated with `ttl=0`:
...
+Starting Rancher v2.4.6, admins can set a global TTL on Kubeconfig tokens. Once the token expires the kubectl command will require the user to authenticate to Rancher.
+
+1. Disable the kubeconfig-generate-token setting in the Rancher API view at `https://<Rancher-Server-IP/v3/settings/kubeconfig-generate-token`. This setting instructs Rancher to no longer automatically generate a token when a user clicks on download a kubeconfig file. The kubeconfig file will now provide a command to login to Rancher.
+
+2. Edit the setting and set the value to `false`. 
+
+3. Go to setting kubeconfig-token-ttl-minutes in the Rancher API view at `https://<Rancher-Server-IP/v3/settings/kubeconfig-token-ttl-minutes`. By default, kubeconfig-token-ttl-minutes is 960 (16 hours).
+
+4. Edit the setting and set the value to desired duration in minutes.
+_**Note:**_ This value cannot exceed max-ttl of API tokens.(`https://<Rancher-Server-IP/v3/settings/auth-token-max-ttl-minutes`). In Rancher v2.4.6, auth-token-max-ttl-minutes is set to 1440 (24 hours) by default. Starting Rancher v2.4.7, auth-token-max-ttl-minutes is set to 0 (never expire).
```
Comment (cloudnautique) [inline content/rancher/v2.x/en/api/api-tokens/_index.md:45]: call out 2.4.7 was going to pre-setting defaults.

**Review by catherineluse — APPROVED** [review]
LGTM

**Pull request merged**